packaging/nnstreamer.spec

```diff
@@ -40,6 +40,12 @@ BuildRequires: cairo-devel
 BuildRequires: opencv-devel
 # For './testAll.sh' time limit.
 BuildRequires: procps
+# for tensorflow
+%ifarch x86_64 aarch64
+BuildRequires: protobuf-devel >= 3.4.0
+BuildRequires: tensorflow
+BuildRequires: tensorflow-devel
+%endif
```

Comment:
ADD "Tensorflow Disable" to ```%%cmake``` statement if it's not x86_64 or aarch64.

Reply:
it's applied